### PR TITLE
Feature/run post process

### PR DESCRIPTION
**Pull request by bd4**
The FOB format has changed, with working_dir moving up to the fob dict with optional per-run override, so campaigns will need to be re-generated. The cheetah spec format has some new options but is backward compat, although the 'post-process' code name is now reserved.

### CODE REVIEW

File: examples/pi1M.txt [hunk 1]
```diff
@@ -0,0 +1 @@
...768017359101557338153772001952444543620071848475663415407442328621060997613243487548847434539665981338717466093020535070271952983943271425371155766600025784423031073429551533945060486222764966687624079324353192992639253731076892135352572321080889819339168668278948281170472624501948409700975760920983724090074717973340788141825195842598096241747610138252643955135259311885045636264188300338539652435997416931322894719878308427600401368074703904097238473945834896186539790594118599310356168436869219485382055780395773881360679549900085123259442529724486666766834641402189915944565309423440650667851948417766779470472041958822043295380326310537494883122180391279678446100139726753892195119117836587662528083690053249004597410947068772912328214304635337283519953648274325833119144459017809607782883583730111857543659958982724531925310588115026307542571493943024453931870179923608166611305426253995833897942971602070338767815033010280120095997252222280801423571094760351925544434929986767817891045559063015953809761875920358937341978962358931125983902598310267193304189215109689156225069659119828323455503059081730735195503721665870288053992138576037035377105178021280129566841984140362872725623214428754
```
Comment (kshitij-v-mehta) [inline examples/pi1M.txt]: Accidental add?

Comment (bd4) [reply examples/pi1M.txt]: It's used by the pi example post processing script, which counts how many correct digits were generated by the run. Maybe not the most instructive example, but useful for testing.

**Comment by kshitij-v-mehta** [comment]
Does this preserve the order of the fob components in fobs.json?

**Comment by bd4** [comment]
Order is always preserved in the fob passed to workflow and the subprocesses are launched in that order, but without sleep_after, the order they actually execute in is a race, since it's async. With sleep_after 1+ seconds, the chances of them running out of order should be negligible.

**Comment by kshitij-v-mehta** [comment]
I ran a small test, and the fob components are listed in a different order in fobs.json. The heat transfer spec file lists stage_write before heat, but in fobs.json, they are in reverse order. This was fixed recently, so I just want to make sure the modification is carried forward

**Comment by kshitij-v-mehta** [comment]
> The FOB format has changed, with working_dir moving up to the fob dict with optional per-run override

why do we need a per-run override

**Comment by bd4** [comment]
Oops I didn't rebase that particular change, I will do that. It would not get lost on a merge, but I like to keep linear history when the rebase is trivial.

Re per-run override, that is not currently needed by cheetah, but was easy to support and could be useful in future, for cheetah or swift. Having part of the app run in a subdir in particular could be useful.

**Pull request merged**